webapp/src/App.tsx

```diff
@@ -19,6 +20,7 @@ function App() {
         >
           Learn React
         </a>
+        <Icon type={IconType.LOGIN} style={{ fill: 'green' }} />
```

Comment:
I think we should define an interface for colors and not passing just `css` :(

Reply:
Here you mean like an enum for colors? Or we could pass hex values?